queries/c_sharp/highlights.scm

```diff
@@ -449,9 +531,14 @@
   "partial"
   "sealed"
   "virtual"
+  "global"
+  (modifier)
 ] @keyword.modifier
 
-(parameter_modifier) @operator
+(modifier) @keyword.modifier
```

Comment:
This is covered above (right now some modifiers are highlighted 3 times); also `"override"` is highlighted as `@keyword` above but also covered in `(modifier)` so highlighted as a modifier later